### ISSUE TITLE
About the hyperparameter setting.

### ISSUE THREAD

**Issue by SuperSaiki** [zequnl/CMAML#4]
Hi @zequnl, 
I followed the default hyper-parameter setting during training.  But I tried many times and the best validation loss I could get is around 96 at the 36-th meta iteration during the training process. But I found that the note in train.sh says that the loss should be around 45 at 55-th meta-iteration. So I wonder whther there is something wrong about the hyper-parameter setting.
